src/app/scripts/cac/pages/cac-pages-map.js

```diff
@@ -179,10 +190,39 @@ CAC.Pages.Map = (function ($, Handlebars, _, moment, MapControl, Routing, MockDe
         }
     }
 
-    function fetchIsochrone() {
-        mapControl.fetchIsochrone().then(function (destinations) {
-            setDestinationSidebar(destinations);
-        });
+    /**
+     * Set user preferences before fetching isochrone.
+     */
+    function clickedExplore() {
+        var exploreMinutes = $('#exploreTime').val();
+        var mode = $('#exploreModeSelector').val();
+
+        // TODO: add date/time selector to 'explore' extra options panel?
+        var when = moment();
+
+        // store search inputs to preferences
+        UserPreferences.setPreference('method', 'explore');
+        UserPreferences.setPreference('originText', $('#exploreOrigin').val());
```

Comment:
The originText is being set, but it looks like the actual origin coordinates aren't. This would lead to a scenario where when the page is reloaded, the text will be set, but it will be using a different set of stored coordinates (probably from when it was stored on the home page).